sdk/appcenter-analytics/build.gradle

```diff
@@ -2,5 +2,5 @@ project.description = 'This package contains functionalities to collect session,
 evaluationDependsOn(':sdk')
 
 dependencies {
-    compile project(':sdk:appcenter')
+    implementation project(':sdk:appcenter')
```

Comment:
api, we don't want people to have to include core explicitly.
Same comment for others.